README.md

```diff
@@ -15,12 +13,43 @@ NNFusion is a flexible and efficient DNN compiler that can generate high-perform
 - Compatible with both hand-crafted kernel code and vendor-provided libraries
 - Customized optimization support through directly rewriting the generated human-readable code
 
-### Getting Started
-To try NNFusion, please read the tutorial on [Compile a Tensorflow model with NNFusion](https://github.com/microsoft/nnfusion/wiki/Compile-a-Tensorflow-model-with-NNFusion).
-
-### Build and Test
-For more detailed building and testing information, please read the Build Guide.
+## Get Started
+### Quick Start with Docker Image
+For end users, simply use docker to compile your model and generate high-performance executable.
+NNFusion supports and is tested on Ubuntu 16.04 and 18.04 with a CUDA GPU equipped. You should install nvidia-docker on your device to do the following steps.
+1. Pull docker image
+`docker pull nnfusion/cuda:10.2-cudnn7-devel-ubuntu18.04`
+2. Run docker container with the given image
+```
+docker run -t --name [YOUR_CONTAINER_NAME] -d nnfusion/cuda:10.2-cudnn7-devel-ubuntu18.04
+docker start [YOUR_CONTAINER_NAME]
+docker exec -it [YOUR_CONTAINER_NAME] bash
+```
+3. Put your model in the container
+In host, you can use `docker cp host_path [YOUR_CONTAINER_NAME]:container_path` to copy your model into the container, or use `docker run -t -i -v <host_dir>:<container_dir>` to map the host dir to the container.
+We will use a simple TensorFlow LSTM inference model as an example. You can download a frozen version from our model zoo:
+`wget https://nnfusion.blob.core.windows.net/models/tensorflow/frozen_lstm_l8s8h256_bs1.pb`
+4. Compile Model
+When model is prepared, we can compile model in the container and run it to see the performance.
+```
+cd root
+nnfusion path/[YOUR_MODEL_FILE]
+```
+5. Build and Run Compiled Model
+```
+cd root/nnfusion_rt/cuda_codegen
+cmake. && make -j
+./main_test
+```
+6. The output of NNFusion should be Tensors with value and model iteration times.
+For more detailed information on NNFusion usage, please refer to [NNFusion Usage](https://github.com/microsoft/nnfusion/wiki/3.-Compile-a-Tensorflow-model-with-NNFusion).
 
+### Build from Source Code
+Researchers or contributors who want to do more research on optimizing model compilation, you can build NNFusion from source code.
+To build from source code, please read the following documents:
+1. Read [Before Started](https://github.com/microsoft/nnfusion/wiki/1.-Before-Started) page to see supported CUDA GPUS and required libs. 
+2. Read [Build Guide](https://github.com/microsoft/nnfusion/wiki/2.-Build-Guide) for more information on how to build and install NNFusion in your native system or in the docker container.
+3. After building and installing NNFusion, please refer to [Compile Guide and Tool Usage](https://github.com/microsoft/nnfusion/wiki/3.-Compile-a-Tensorflow-model-with-NNFusion) to learn how to compile or optimize a DNN model.
 
 # Contributing
 
```

Comment:
Shall we add a section for citation of the work? (i.e. the paper) refer to: 
https://github.com/Microsoft/SPTAG#references